templates/nextflow-s3-template.yaml

```diff
@@ -95,62 +98,89 @@ Resources:
               - !Sub arn:aws:s3:::${Bucket}
               - !Sub arn:aws:s3:::${Bucket}/*
 
-          - Sid: ReadWriteAccess
-            # gives grantees read/write write access to the bucket with required --acl
-            Effect: Allow
-            Principal:
-              AWS: !Split
-                - ","
-                - !Join
-                    - ","
-                    - - Fn::ImportValue: !Sub "${AWS::Region}-nextflow-forge-service-user-ServiceUserArn"
-                      - !Join
-                          - ","
-                          - !Ref ReadWriteAccessArns
-            Action:
-              - "s3:*Object*"  
-            Condition:
-              StringEquals:
-                s3:x-amz-acl: bucket-owner-full-control
-            Resource:
-              - !Sub arn:aws:s3:::${Bucket}/*    
+          - !If
+            - NoReadWriteAccessArns
```

Comment:
I think we can unconditionally grant read/write access to the Tower Forge IAM user since policy statements should complement each other. In other words, if two different statements grant two different principals the same permissions, they will both apply. So, we wouldn't need the `NoReadWriteAccessArns` condition.

Reply:
Done! Added `Sid: ForgeUserFullAccess`